app/views/users/_form.html.erb

```diff
@@ -1,28 +1,52 @@
 <%= form_tag path, method: method do %>
 
```

Comment:
I think there must be a better way to reuse this partial (maybe `form_for`?). If no one has ideas right away we can create an issue to revisit it in the future.

Reply:
Brian or Tim said something about a ```form_for``` .  So that would probably be something we would want to implement at some point

Reply:
Hopefully this will make more sense after tomorrow's class...

Reply:
Yay Monday!  I'm going to resolve this conversation because it looks like we might've just tabled it for tomorrow's class.